packages/shylu/shylu_dd/frosch/test/TestLaplacian_Epetra/main.cpp

```diff
@@ -67,7 +67,11 @@ typedef unsigned UN;
 typedef double SC;
 typedef int LO;
 typedef int GO;
-typedef Kokkos::Compat::KokkosSerialWrapperNode EpetraNode; // Hier Default verwenden???
+#ifdef KOKKOS_HAVE_SERIAL
+typedef Kokkos::Compat::KokkosSerialWrapperNode EpetraNode;
+#else
+typedef Kokkos::Compat::KokkosOpenMPWrapperNode EpetraNode;
+#endif
 typedef EpetraNode NO;
 
 using namespace std;
```

Comment:
It's generally bad practice to do `using namespace std;` at the global scope.  It's acceptable for a unit test .cpp file, but never OK for a header file.